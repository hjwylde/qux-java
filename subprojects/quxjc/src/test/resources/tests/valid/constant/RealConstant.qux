real A is 5.0
real B is 5.0
real C is -5.0

real f(real a):
    return a + A + 0.5

real g(real A):
    return A + 0.5


void main([str] args):
    print A
    print B
    print C
    print A == B
    print A == C
    print B == C
    print f(C)
    print g(C)

