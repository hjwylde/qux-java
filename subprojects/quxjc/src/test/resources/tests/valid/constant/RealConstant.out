5/1
5/1
-5/1
true
false
false
1/2
-9/2
